### DISCUSSION TITLE
How to Configure EJBCA to override subjectAlternativeName in Certificates?

### DISCUSSION THREAD

**Discussion by prajna119** [Keyfactor/ejbca-ce#720] [Q&A] [answered]
Can someone guide us on what configuration changes are needed in EJBCA server to ensure that the subjectAlternativeName in the certificate matches the value provided in the request?

We’re already including the desired value (0005B9519090.cs.com), but the field doesn’t seem to be overridden as expected.

![image](https://github.com/user-attachments/assets/aa2df898-b975-47d1-985b-7c36f358e74d)

**Comment by primetomas** [comment, marked as answer]
It's documented here: https://docs.keyfactor.com/ejbca/latest/certificate-profile-fields#id-(latest)CertificateProfileFields-AllowExtensionOverrideAllowExtensionOverride
